alu/ext/columns.lisp

```diff
@@ -1,19 +1,19 @@
 (module ext)
 
 (defcolumns
-    STAMP
-    (OLI :binary)
-    CT
-    INST
+    (STAMP   :i64)
```

Comment:
Why not :i32 as for all other stamps ?

Reply:
No reason, just a quick fix. `:i32` works fine, too.